src/main/cpp/t07_ip.cpp

```diff
@@ -29,10 +29,48 @@
 
 #include "t07_ip.h"
 #include <iostream>
+#include<string>
 
 
 using namespace std;
 
 int t07_ip() {
+  string str,sub;
+  int probel=0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'probel' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4513001)